nativeaot-sb/test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+set -x
+
+RID_ARG="/p:RuntimeIdentifier=$(../runtime-id --sdk)"
+
+for DEPENDENCY in NO_DEPS DEP_CRYPTO DEP_ZLIB DEP_BROTLI
```

Comment:
Other dependencies (eg, `libunwind`, `rapidjson`) are mandatory for the runtime (if the runtime uses the system version of those libraries). So the program will probably fail to start if they are missing. Should we document those here?

Reply:
I've added a comment.